docs/learn/encyclopedia/transactions-specialized/channel-accounts.mdx

```diff
@@ -341,7 +338,7 @@ for channelIndex, channels in enumerate(channelAccounts):
       asset = Asset.native()
     )
 
-  transaction = transaction.set_timeout(3600).build()
+  transaction = transaction.build()
```

Comment:
Should leave a timeout, but before building

Reply:
I was thinking a lot about this since it's an advanced guide that is already getting fairly lengthy. My idea was to make the code just a little easier to comprehend. But I now see more what you mean and agree that real transactions should always specify a timeout. ⏱️

Upon reflection, I agree with consistent inclusion along the lines of trying to write pages so well that even beginners can comprehend them, as championed by innovators like Garand. 💡 Accordingly, it makes sense to (potentially) introduce them to good hygiene, like including timeouts.[^ct] I chose `3600` since it seemed like a reasonable hot submission threshold, but do let me know if you have an opinionated preference.

[^ct]: _See, e.g.,_ how I've tried to do this through limited, timely, relevant hyperlinks to other docs pages. _Compare_ with other community members that expertly explain complex items with grace using prudent [images](https://medium.com/stellar-community/daos-donts-1-the-dao-28a543eeb711) or [videos](https://www.youtube.com/playlist?list=PLWUFvhKuc_5u1sQsz-FAmRgFf9HEhyqj-). 🙌🏽

Reply:
Implemented in 06183cc910016dfd8cc4fe227474da927f5595f0 🙌🏻🫱🏻‍🫲🏼